lib/pg_rls/tenant.rb

```diff
@@ -9,6 +9,7 @@ def switch(resource)
         find_tenant(resource)
         connection_adapter.connection.execute(format('SET rls.tenant_id = %s',
                                                      connection_adapter.connection.quote(tenant.tenant_id)))
+        @fetch = nil
```

Comment:
###  Last comment sorry for being picky but since this won't have any test created yet I must be careful with our changes 😓, if you can make this change this weekend I'll approve it and we can merge it, so I can generate a new version :) BTW, can you change the version so it would include your changes 😉 

Please move this to the top, so if anything fails we will guarantee the `@fetch `was reseted  

```ruby
# lib/pg_rls/version.rb

module PgRls
  VERSION = '0.0.1.4.2'
end
```
sorry I fail to mention this before